src/modules/Elsa.JavaScript/Services/JintJavaScriptEvaluator.cs

```diff
@@ -73,12 +74,13 @@ private async Task<Engine> GetConfiguredEngine(Action<Engine>? configureEngine,
 
         // Create variable & input setters and getters for each variable.
         CreateMemoryBlockAccessors(engine, context);
-        
 
         engine.SetValue("isNullOrWhiteSpace", (Func<string, bool>)(value => string.IsNullOrWhiteSpace(value)));
         engine.SetValue("isNullOrEmpty", (Func<string, bool>)(value => string.IsNullOrEmpty(value)));
         engine.SetValue("parseGuid", (Func<string, Guid>)(value => Guid.Parse(value)));
         engine.SetValue("toJson", (Func<object, string>)(value => Serialize(value)));
+        engine.SetValue("getShortGuid", (Func<string, string>)(value => Regex.Replace(Convert.ToBase64String(Guid.NewGuid().ToByteArray()), "[/+=]", "")));
+        engine.SetValue("getFullGuid", (Func<string, Guid>)(value => Guid.NewGuid()));
```

Comment:
Let's rename this to `getGuid` as the counterpart for `parseGuid` (as opposed to `parseFullGuid`)?